pkgs/tools/networking/dd-agent/default.nix

```diff
@@ -26,61 +33,158 @@ let
     # due to flake8
     doCheck = false;
   };
+  version = "5.13.2";
+
+  integrations = fetchFromGitHub {
+    owner = "datadog";
+    repo = "integrations-core";
+    rev = version;
+    sha256 = "1nbjmkq0wdfndmx0qap69h2rkwkkb0632j87h9d3j99bykyav3y3";
+  };
+
+  jmxArtifact = fetchurl
+    { url = "http://dd-jmxfetch.s3.amazonaws.com/jmxfetch-0.13.0-jar-with-dependencies.jar";
+      sha256 = "02rfn17q42kxiv2sv306ibdhvi8l4sbjdwhh1nwdlxw46pqh4ks1";
+    };
 
 in stdenv.mkDerivation rec {
-  version = "5.11.2";
   name = "dd-agent-${version}";
 
   src = fetchFromGitHub {
     owner  = "datadog";
     repo   = "dd-agent";
     rev    = version;
-    sha256 = "1iqxvgpsqibqw3vk79158l2pnb6y4pjhjp2d6724lm5rpz4825lx";
+    sha256 = "0x2bxi70l2yf0wi232qksvcscjdpjg8l7dmgg1286vqryyfazfjb";
   };
 
   buildInputs = [
     python
     unzip
     makeWrapper
-    pythonPackages.requests
-    pythonPackages.psycopg2
-    pythonPackages.psutil
```

Comment:
I think that defining the inclusion of package dependencies only in the configuration of nixos modules goes against the design of nix.  Nix doesn't exist only in the context of nixos and never has.  For those not using nixos (such as myself), the removal of this dependency effectively breaks the dd-agent package since the basic components such as network and core no longer work.  That has the effect of breaking a working package for existing users.  I would ask that this not be merged without finding a way not to do that.